### PR TITLE
add LatexString overload, add test

### LINKED ISSUE #474: [BUG] keyword arguments do not match for latex()
**Describe the bug**

The `latex()` function has two keyword arguments `valign` and `halign`. Specifying them in a function call to `latex` causes a `MethodError`.

**To Reproduce**

1. Julia Version (i.e. output of `julia -v`): 1.7.2

2. Operating system (Mac, Linux, Windows): Ubuntu Linux 20.04

3. Javis version (i.e output of `] status Javis` in the REPL) v0.8.0

4. Minimum working code example that led to bug:
```
using Javis
using LaTeXStrings

function ground(args...)
    background("white")
    sethue("black")
end

vid = Video(300, 200)
Background(1:1, ground)
Object((args...) -> latex(L"0", O, valign = :top, halign = :right))
render(vid; pathname="thing.gif", framerate=1)
```

In [Javis tutorial #4](https://juliaanimators.github.io/Javis.jl/stable/tutorials/tutorial_4/), code was given which specified valign and halign as keyword arguments. The keyword arguments are not available.

**Stacktrace (If Applicable)**
```
ERROR: LoadError: MethodError: no method matching latex(::LaTeXString, ::Point; valign=:top, halign=:right)
Closest candidates are:
  latex(::LaTeXString, ::Point) at ~/.julia/packages/Javis/TCcr1/src/latex.jl:5 got unsupported keyword arguments "valign", "halign"
  latex(::LaTeXString, ::Point, ::Symbol; valign, halign) at ~/.julia/packages/Javis/TCcr1/src/latex.jl:71
  latex(::LaTeXString, ::Point, ::Symbol, ::Symbol) at ~/.julia/packages/Javis/TCcr1/src/latex.jl:6 got unsupported keyword arguments "valign", "halign"
  ...
Stacktrace:
 [1] (::var"#1#2")(::Video, ::Vararg{Any})
   @ Main ~/Documents/Code/learning/julia/using Javis.jl:11
 [2] draw_object(object::Object, video::Video, frame::Int64, origin_matrix::Matrix{Float64}, layer_frames::Nothing)
   @ Javis ~/.julia/packages/Javis/TCcr1/src/Javis.jl:658
 [3] macro expansion
   @ ~/.julia/packages/Javis/TCcr1/src/Javis.jl:408 [inlined]
 [4] macro expansion
   @ ~/.julia/packages/Luxor/Oftfe/src/basics.jl:526 [inlined]
 [5] render_objects(objects::Vector{Javis.AbstractObject}, video::Video, frame::Int64; layer_frames::Nothing)
   @ Javis ~/.julia/packages/Javis/TCcr1/src/Javis.jl:407
 [6] render_objects
   @ ~/.julia/packages/Javis/TCcr1/src/Javis.jl:380 [inlined]
 [7] get_javis_frame(video::Video, objects::Vector{Javis.AbstractObject}, frame::Int64; layers::Vector{Javis.AbstractObject})
   @ Javis ~/.julia/packages/Javis/TCcr1/src/Javis.jl:601
 [8] render(video::Video; framerate::Int64, pathname::String, liveview::Bool, streamconfig::Nothing, tempdirectory::String, ffmpeg_loglevel::String, rescale_factor::Float64, postprocess_frames_flow::typeof(identity), postprocess_frame::type
of(Javis.default_postprocess))
   @ Javis ~/.julia/packages/Javis/TCcr1/src/Javis.jl:288
 [9] top-level scope
   @ ~/Documents/Code/learning/julia/using Javis.jl:12
in expression starting at /home/razetime/Documents/Code/learning/julia/using Javis.jl:12
```

### PR DESCRIPTION
**Pull request by razetime**
Closes #474.

After some discussion in the Javis.jl Zulip chat, I added an overload for the `latex` function with `valign` defaulting to `:top` and `halign` defaulting to `:left`.

A single regression test was added in `svg.jl` to make sure that this overload was working correctly.

### CODE REVIEW

File: test/svg.jl [hunk 1]
```diff
@@ -54,6 +54,7 @@ end
     Object((args...) -> latex(L"8", O + Point(20, 20), :bottom, :right))
     Object((args...) -> latex(L"8", O + Point(20, 20), :bottom, :left))
     Object((args...) -> latex(L"8", O + Point(20, 20), :top, :right))
+    Object((args...) -> latex(L"8", O + Point(20, 20), valign = :top, halign = :right))
```
Comment (Wikunia) [inline test/svg.jl:57]: The line before should be removed, right?

Comment (razetime) [reply test/svg.jl:57]: I am not sure why it should be removed. i was thinking the explicit arguments were a different test.

Comment (Wikunia) [reply test/svg.jl:57]: Now it's drawing the same text twice at the same position.

Comment (razetime) [reply test/svg.jl:56]: Ok, removed it.

**Comment by Wikunia** [comment]
Thanks :)

**Pull request merged**